recipe/meta.yaml

```
@@ -1,36 +1,43 @@
-{% set name = "WebOb" %}
-{% set version = "1.8.7" %}
-{% set sha256 = "b64ef5141be559cfade448f044fa45c2260351edcb6a8ef6b7e00c7dcef0c323" %}
+{% set name = "webob" %}
+{% set version = "1.8.9" %}
 
 package:
-  name: {{ name|lower }}
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ad6078e2edb6766d1334ec3dee072ac6a7f95b1e32ce10def8ff7f0f02d56589
 
 build:
-  noarch: python
+  skip: true # [py<39]
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   host:
-    - pip
     - python
-    - setuptools
-
+    - pip
+    - wheel
```

https://github.com/Pylons/webob/blob/1.8.9/setup.py#L3
```suggestion
    - wheel
    - setuptools
```